lib/fittings/swagger_locals.js

```diff
@@ -0,0 +1,45 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+var injector = require('../../index.js').injector;
+var _ = injector.get('_');    // jshint ignore:line
+var Promise = injector.get('Promise');    // jshint ignore:line
+var debug = require('debug')('swagger:locals'); // jshint ignore:line
+var Error = injector.get('Errors'); // jshint ignore:line
+
+var skuParamHandler = function(context, value) {
```

Comment:
Something like another layer of dependency injection or something like the router controllers?  Either way, seems like there would be some value in supporting that later.